tests/SprykerSdkTest/Spryk/Integration/Zed/Persistence/AddZedPersistenceRepositoryTest.php

```diff
@@ -75,10 +74,9 @@ public function testAddsZedRepositoryFileThatExtendsSameCoreBaseClass(): void
             '--mode' => 'project',
         ]);
 
-        $targetClassFilePath = $this->tester->getProjectModuleDirectory('FooBar', 'Zed') . 'Persistence/FooBarRepository.php';
-
-        $this->assertFileExists($targetClassFilePath);
-
-        $this->tester->assertClassOrInterfaceExtends($targetClassFilePath, 'Spryker\Zed\FooBar\Persistence\FooBarRepository');
+        $this->tester->assertClassOrInterfaceExtends(
+            ClassName::PROJECT_ZED_REPOSITORY,
+            'Spryker\Zed\FooBar\Persistence\FooBarRepository',
```

Comment:
Why are all those class name strings not replaced with a constant as well? Please update all string references with constants.